frontend/packages/console-shared/src/utils/validation/validation.ts

```diff
@@ -54,7 +60,11 @@ export const validateDNS1123SubdomainValue = (
   const forbiddenCharacters = new Set<string>();
   const validationSentences = [];
 
-  if (value.length > DNS1123_MAX_LENGTH) {
+  if (min && value.length < min) {
+    validationSentences.push(getStringTooShortErrorMsg(min));
+  } else if (max && value.length > max) {
+    validationSentences.push(getStringTooLongErrorMsg(max));
+  } else if (value.length > DNS1123_MAX_LENGTH) {
```

Comment:
we can default max to DNS1123_MAX_LENGTH and then we can shorten these two cases under one. Plus we can delete DNS1123_TOO_LONG_ERROR constant

Reply:
Very nice! I wish I had thought of that. =) Done!